packages/flutter/lib/src/services/system_channels.dart

```diff
@@ -159,6 +159,11 @@ class SystemChannels {
   ///    second argument is a [String] consisting of the stringification of one
   ///    of the values of the [TextInputAction] enum.
   ///
+  ///  * `TextInputClient.requestExistingInputState`: The embedding may have
```

Comment:
Let's document what should happen if there's nothing to send.

Reply:
Done